mwdb/web/src/components/ShowObject/PushAction.js

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useState } from "react";
+import { connect } from "react-redux";
+
+import { faGlobe } from '@fortawesome/free-solid-svg-icons'
+
+import api from "@mwdb-web/commons/api";
+import { ObjectContext } from "@mwdb-web/commons/context";
+import { ObjectAction } from  "@mwdb-web/commons/ui";
+import { ConfirmationModal } from "@mwdb-web/commons/ui";
+
+function PushAction(props) {
+    const context = useContext(ObjectContext);
+    const [isPushModalOpen, setPushModalOpen] = useState(false);
+    const [remoteName, setRemoteName] = useState("");
+
+    async function pushRemote() {
+        let type = context.object.type;
+        if (type !== "file") {
+            type = type.split("_")
+            type = type[1]
+        }
```

Comment:
Clever 😄 But not very readable/grepable. I would prefer some explicit mapping like:
```javascript
({
    "file": "file",
    "static_config": "config",
    "text_blob": "blob"
})[context.object.type]
```